eval/eval.py

```diff
@@ -1782,7 +1791,7 @@ def process(fpath_and_category: Tuple[str, str]) -> Dict[str, Any]:
                 ]
             )
             # Pass it to GreynirCorrect
-            pg = [list(p) for p in gc_check(text, **options)]
+            pg = [list(p) for p in gc_check(text, rc, **options)]
```

Comment:
Would rc=rc be better here? If the API is being changed, it's better to use explicit keyword-only parameters.

Reply:
Fixed, and also, now rc is optional in the check method